eng/.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
FYI I very much prefer to open `eng` directory in vscode, as if I open entire repo it takes forever for `vscode` search to index all files, and so on.

However, I am unsure how to approach avoiding deduping settings. Looks like VSCode does not support "includes":

- https://github.com/microsoft/vscode/issues/15909

Reply:
I open repo root in VSCode and perf seems fine, but I don't mind duplicating a few settings.

Reply:
Does opening the eng folder work with the root package-lock.json?  Because we only want a single package-lock.json in repo root, not a lockfile per folder.

Reply:
I think it does, but not sure how I would now it does not work. In any case, I suggest we figure out the .vscode settings situation in a follow-up PR.